aten/src/ATen/native/mkldnn/xpu/detail/Utils.cpp

```diff
@@ -18,6 +18,13 @@ dnnl::memory make_onednn_memory(
       ptr == nullptr ? DNNL_MEMORY_ALLOCATE : ptr);
 }
 
+dnnl::memory make_onednn_memory(
+    dnnl::memory::desc md,
+    dnnl::engine& engine,
+    const void* ptr) {
+  return make_onednn_memory(md, engine, (void*)ptr);
```

Comment:
Since the ptr is a `const void*`, what's the motivation to cast it to `void*` forcelly?

Reply:
Reduce the duplicated code.

make_onednn_memory(md, engine, void* ptr)
make_onednn_memory(md, engine, const void* ptr) -> make_onednn_memory(md, engine, void* ptr)